src/pkg/cli/tail.go

```diff
@@ -112,16 +123,16 @@ func Tail(ctx context.Context, client client.Client, service, etag string, since
 
 	ctx, cancel := context.WithCancel(ctx)
```

Comment:
unable to make cancel a defer as there is some interaction with the other defers and the order of calls. It's simpler to keep the two location where it is called directly.

Reply:
actually that cancel triggered a Golang lint error, which suggested to always defer: #374